web-access-app/src/main/resources/search.properties

```
@@ -1,11 +1,11 @@
 search.results.maxPerPage=2000
 #Fields which are searchable via keyword searches
-search.field.searchable=ID,DEFAULT_INDEX,TITLE_INDEX,CONTRIBUTOR_INDEX,SUBJECT_INDEX,IDENTIFIER\
+search.field.searchable=ID,DEFAULT_INDEX,TITLE_INDEX,CREATOR_INDEX,CONTRIBUTOR_INDEX,SUBJECT_INDEX,IDENTIFIER\
```

don't think CREATOR_INDEX exists
```suggestion
search.field.searchable=ID,DEFAULT_INDEX,TITLE_INDEX,CONTRIBUTOR_INDEX,SUBJECT_INDEX,IDENTIFIER\
```

---

Whoops, it did exist at one point in my pr.